components/mpas-ocean/src/mode_forward/mpas_ocn_time_integration_lts.F

```diff
@@ -733,24 +711,30 @@ subroutine ocn_time_integrator_lts(domain,dt)!{{{
        do ie = 1, nEdgesInLTSRegion(1,2)
           iEdge = edgesInLTSRegion(1,2,ie)
           normalVelocityFirstStage(:,iEdge) = normalVelocityNew(:,iEdge)
-          normalVelocityTendSum1st(:,iEdge) = normalVelocityTendSum1st(:,iEdge) + normalVelocityTend(:,iEdge)
+          normalVelocityTendSum1st(:,iEdge) = ( 1.0_RKIND / (3.0_RKIND * dt * weightTendSum1st) * normalVelocityCur(:,iEdge) & 
+                                                + normalVelocityTendSum1st(:,iEdge) + normalVelocityTend(:,iEdge) ) &
+                                              * (1.0_RKIND - wettingVelocityFactor(:, iEdge))
        end do
        ! --- interface layer 2 (correction)
        do ie = 1, nEdgesInLTSRegion(2,2)
           iEdge = edgesInLTSRegion(2,2,ie)
-          normalVelocityTendSum1st(:,iEdge) = normalVelocityTendSum1st(:,iEdge) + normalVelocityTend(:,iEdge)
+          normalVelocityTendSum1st(:,iEdge) = ( 1.0_RKIND / (3.0_RKIND * dt * weightTendSum1st) * normalVelocityCur(:,iEdge) & 
+                                                + normalVelocityTendSum1st(:,iEdge) + normalVelocityTend(:,iEdge) ) &
+                                              * (1.0_RKIND - wettingVelocityFactor(:, iEdge))
        end do
 
        ! --- fine (soln advancement)
        do ie = 1, nEdgesInLTSRegion(1,1)
           iEdge = edgesInLTSRegion(1,1,ie)
-          normalVelocitySecondStage(:,iEdge) =  weightOld * normalVelocityCur(:,iEdge) + weightNew * normalVelocityFirstStage(:,iEdge) &
-                                               + weightTend *  dtFine * normalVelocityTend(:,iEdge) !soln update for the fine
+          normalVelocitySecondStage(:,iEdge) = ( weightOld * normalVelocityCur(:,iEdge) + weightNew * normalVelocityFirstStage(:,iEdge) &
+                                                + weightTend *  dtFine * normalVelocityTend(:,iEdge) ) & !soln update for the fine
+                                               * (1.0_RKIND - wettingVelocityFactor(:, iEdge))
        end do
        do ie = 1, nEdgesInLTSRegion(1,3)
           iEdge = edgesInLTSRegion(1,3,ie)
-          normalVelocitySecondStage(:,iEdge) =  weightOld * normalVelocityCur(:,iEdge) + weightNew * normalVelocityFirstStage(:,iEdge) &
-                                                + weightTend *  dtFine * normalVelocityTend(:,iEdge) !soln update for the fine
+          normalVelocitySecondStage(:,iEdge) = ( weightOld * normalVelocityCur(:,iEdge) + weightNew * normalVelocityFirstStage(:,iEdge) &
+                                                + weightTend *  dtFine * normalVelocityTend(:,iEdge) ) & !soln update for the fine
+                                               * (1.0_RKIND - wettingVelocityFactor(:, iEdge))
```

Comment:
Same here, I would think you only need to scale the last term by `1-wettingVelocityFactor`

Reply:
if we do that then the velocity would not be set to zero since the old velocity is not limited